databroker/core.py

```diff
@@ -647,6 +666,28 @@ def _load(self):
                 getshell=True,
                 catalog=self)
 
+    def get(self):
```

Comment:
does this need to be able to pass `arg, kwargs` trough?

Reply:
When I wrote this I intentionally did _not_ support it, but now I think that was based on a confused notion. Added in fcc489f86e0d78991c8193b442dbe19f8e9a6e1e.